src/controllers/UsersController.js

```diff
@@ -0,0 +1,74 @@
+import models from '../db/models';
+import TokenAuthenticate from '../helpers/tokenAuthentication';
+import Response from '../helpers/response';
+const { User, Profile } = models;
```

Comment:
Why do you have to destructor models before other imports? Do you have a reason for this? If not can you take it down below other imports

Reply:
An oversight, thanks for pointing that out. I will change it